mruby/tic_default.rb

```diff
@@ -0,0 +1,41 @@
+MRuby::Build.new do |conf|
+  conf.toolchain
+
+  conf.gembox 'default'
+
+  conf.enable_bintest
+  conf.enable_test
+end
+
+MRuby::CrossBuild.new('target') do |conf|
```

Comment:
Could the `mruby` config files live in `src/api/mruby` or somewhere other than the root? Having it in the root seem unorganized. Thanks!

Reply:
In the case of the other scripting languages, the dependencies are packed in. `rake` or `yacc` won't be available on a few different build systems. Curious if we could get it to build having needing them in there :thinking: